src/views/orb-map-view.ts

```diff
@@ -226,6 +233,10 @@ export class OrbMapView<N extends INodeBase, E extends IEdgeBase> implements IOr
     this._canvas.outerHTML = '';
   }
 
+  update(): void {
+    this.render();
+  }
```

Comment:
Hmm, this is maybe a bit tricky, because on external API (orb view), there is an `.update()` call because of an observer. Is there a way to handle this better? Would an observer with a callback do the trick or?

Reply:
Is there anything we can do there? Do you have any ideas?

Reply:
Do you maybe have an idea how to implement this?

Reply:
Currently I am seeing two ideas, but there can be more for sure:
1) Have a proxy class that is an actual listener, but also gives you a callback way to respond to updates. Then view can initiate that proxy class where `update` will be called, and update calls a callback that is registered privately from the view. Then `update` is not exposed - easier
2) Add callback to the observer pattern, so listener can either have a callback or extending the `update` to get calls for notifications  - harder